utilities/descending_consensus/descending_consensus.py

```diff
@@ -0,0 +1,65 @@
+import csv
```

Comment:
This script seems to exist in two different places, utilities/consensus/descending_consensus and utilities/descending_consensus. Should one be deleted?